### PR TITLE
Adds a metrics capabilities page for Pomerium Zero 

### PR DESCRIPTION
**Pull request by ZPain8464**
This PR:
- Rewrites the content on https://www.pomerium.com/docs/capabilities/metrics.
- Rewrites the front matter description and keywords.
- Removes the `enterprise` class name from sidebars.js.

This PR is contingent on https://github.com/pomerium/documentation/pull/1621. Once the Prometheus metrics guide is merged, I'll merge changes in this PR and link to the new Configure Metrics guide.

Related to https://github.com/pomerium/documentation/issues/1618.

### CODE REVIEW

**Review by wasaga — COMMENTED** [review]
1. This page has 80% overlap with Capabilities > Reports
2. I suggest we merge them and call something like "Traffic Monitoring"

File: content/docs/capabilities/metrics.md [hunk 1]
```diff
@@ -1,79 +1,118 @@
...
+
+### Bytes sent and received
+
+The **Bytes sent** and **Bytes received** charts display the average amount of bytes sent and received over the span of an hour.
+
+![A chart displaying bytes sent and received in Pomerium Zero](./img/metrics/zero-bytes-sent-received.png)
+
+## Metrics in Pomerium Enterprise
+
+In Pomerium Enterprise, you must configure metrics before you can view them. Metrics are not enabled by default, and are not required to run Pomerium Enterprise. See the **Configure Metrics** guide to learn how to enable metrics in your Enterprise deployment.
```
[the remark this answers is not in the document]
Comment (ZPain8464) [reply content/docs/capabilities/metrics.md:116]: I have to merge https://github.com/pomerium/documentation/pull/1621 first.

File: content/docs/capabilities/metrics.md [hunk 2]
```diff
@@ -1,79 +1,266 @@
...
 ---
 
-Pomerium Enterprise uses Prometheus as a metrics collection back-end. You can configure Pomerium and the Console to talk to an existing Prometheus server, or configure the embedded Prometheus backend. This guide assumes you're running both Pomerium and Pomerium Enterprise on localhost `127.0.0.1`.
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem';
 
-:::tip
+Metrics in Pomerium provide observability and monitoring data from your Pomerium deployment. Use metrics to review traffic and its effects on your system.
 
-For production deployments, we suggest using a dedicated Prometheus instance.
+Metrics are available only in Pomerium Zero and Pomerium Enterprise deployments.
```
Comment (wasaga) [inline content/docs/capabilities/metrics.md:16]:
```suggestion
Pomerium exposes standard Prometheus metrics related to its operation. 
Open source users need [configure](/docs/reference/metrics) metrics collection and visualization manually. 
Pomerium Zero and Pomerium Enterprise have simplified collection and visualization built-in. 
```

File: content/docs/capabilities/metrics.md [hunk 3]
```diff
@@ -1,79 +1,266 @@
...
+<TabItem value="enterprise" label="Enterprise">
+
+The **Request Size** chart organizes requests based on their request size measured in kilobytes (KB).
+
+![A chart displaying response size in Pomerium Enterprise](./img/metrics/enterprise-request-size.png)
+
+</TabItem>
+</Tabs>
+
+## Changesets and Deployments
```
Comment (wasaga) [inline content/docs/capabilities/metrics.md:218]: this section is definitely unrelated to Metrics, and probably has to be moved to its own subsection.

File: content/docs/capabilities/metrics.md [hunk 4]
```diff
@@ -1,79 +1,266 @@
...
-   ```yaml
-   prometheus_url: http://192.168.122.50:9090
-   ```
+In the **Runtime** dashboard, you can monitor how many system resources Pomerium is consuming. Filter by date range, service, and instance.
 
-1. Restart the Pomerium and Pomerium Enterprise services. You should now see route traffic and External Data Source monitoring data in the Enterprise Console:
+![The Runtime Info page in Pomerium Enterprise](./img/metrics/reports-runtime-fullpage.png)
 
-   ![Traffic Data in Pomerium Enterprise](./img/metrics/console-route-traffic.png) ![External Data Source in Pomerium Enterprise](./img/metrics/console-ext-datasource-monitoring.png)
+## Sessions (Enterprise)
```
Comment (wasaga) [inline content/docs/capabilities/metrics.md:256]: This section is unrelated to metrics and need be moved to a separate section.

Comment (ZPain8464) [reply content/docs/capabilities/metrics.md:256]: Same question as above.

File: content/docs/capabilities/metrics.md (hunk 3 above)
Comment (wasaga) [reply content/docs/capabilities/metrics.md:218]: right I think now that we merged _metrics_ content together for zero and enterprise its great. 

Maybe lets call the new section Changesets?
